docs/UserGuide.md

```diff
@@ -149,14 +150,16 @@ In the example above, entering <code>name</code>, or a substring of name like <c
 
 <div style="page-break-after: always;"></div>
 
-After you are done editing, pressing the Enter key on your keyboard brings your cursor back to the command box with your new changes intact. 
+After you are **done editing**, _press the Enter key_ on your keyboard to bring your cursor back to the command box. 
 
 <div></div>
 
-If you wish to revert back to the original value of the field while editing in the textbox, press the Esc key to undo the changes and direct your cursor back to the command box.
+If you **wish to discard changes** while editing in the textbox, _press the Esc key_ to undo the changes and direct your cursor back to the command box.
```

Comment:
i feel like this might be misinterpreted as like all recent changes, rather than the revert being specific to that one field though. Ill clarify the phrasing

Reply:
That's true. Maybe we can mention something about unconfirmed changes? Since other fields' changes would be confirmed.

Reply:
that may bring about the need to define what is confirmed and what is unconfirmed though! u see if u wanna add it in ur PR, ill leave it as is first